### PR TITLE
chinese doc concepts add server descrption

### PR DESCRIPTION
**Pull request by xiangxiong**
1、add server description

### CODE REVIEW

File: pages/zh/lb4/Concepts.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ LoopBack 4 定义了一些组件模块来表达典型的 API 和微微服务应
 ![Key concepts overview diagram](imgs/key-concepts-overview-diagram.png)
 
 - [**应用（Application）**](Application.md)：这是一个重要的类，用于设置你所有的模块组件、控制器、服务器和绑定。Application 类扩展了[上下文（Context）](Context.md) ，并且提供了启动和停止相关服务器的控制方法。
+- [**服务（Server）**]：服务器：入站传输/协议的实现，例如 REST (http, https)、gRPC (http2) 和 graphQL (http, https)。 它通常侦听特定端点（协议/主机/端口），处理传入请求，然后返回适当的响应。
```
Comment (dhmlau) [inline pages/zh/lb4/Concepts.md]: Adding the link to `Server.md`. 
```suggestion
- [**服务（Server）**](Server.md)：服务器：入站传输/协议的实现，例如 REST (http, https)、gRPC (http2) 和 graphQL (http, https)。 它通常侦听特定端点（协议/主机/端口），处理传入请求，然后返回适当的响应。
```

**Review by dhmlau — COMMENTED** [review]
@xiangxiong, thanks for your contribution. Just a minor comment.

**Comment by dhmlau** [comment]
@xiangxiong, could you please also sign the DCO and rebase the branch?  Thanks. 

As a reference, you can sign the commit by using the command: `git commit --amend -s`. See https://loopback.io/doc/en/contrib/code-contrib.html.

File: pages/zh/lb4/Concepts.md (hunk 1 above)
Comment (xiangxiong) [reply pages/zh/lb4/Concepts.md]: OK

**Comment by dhmlau** [comment]
@xiangxiong, thanks for working through this PR. It looks like there are multiple commits in this PR and all need to be signed. Could you please squash the commits?

**Comment by dhmlau** [comment]
@xiangxiong, I've squashed the commits for you. 
Thanks for your contribution. Your PR has landed! 🎉

**Pull request merged**